typescript/radix-sdk/.mocharc.e2e.json

```diff
@@ -0,0 +1,6 @@
+{
+  "import": ["tsx"],
+  "file": ["src/tests/e2e-test.setup.ts"],
+  "spec": ["src/tests/**/*.e2e-test.ts"],
```

Comment:
Does this make sense, while the actual tests are still in `cli/src/tests/radix`?